src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -247,10 +253,19 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
       }
     }
 
-    "Attempting to enable a previously disabled user should return success" in {
-      Post(enablePath, Seq(disabledUser, enabledUser)) ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
-        val enableResponse = responseAs[Map[String, Seq[String]]]
-        assertResult(Map("Success"->Seq(disabledUser), "NoChangeRequired"->Seq(enabledUser))) { enableResponse }
+    "Attempting to enable multiple users in various states should return a map of status lists" in {
+      val assortmentOfUserEmails = Seq(disabledUser, enabledUser, enrolledUser, terminatedUser, dummy1User, dummy2User)
+
+      Post(enablePath, assortmentOfUserEmails) ~> dummyUserIdHeaders(manager) ~> trialApiServiceRoutes ~> check {
+        val enableResponse = responseAs[Map[String, Seq[String]]].map(_.swap)
+
+        assert(enableResponse(List(enabledUser, dummy2User)) === StatusUpdate.NoChangeRequired.toString)
```

Comment:
this test - which relies on a `List` with multiple elements - is fragile in that it is dependent on the order of the users passed in to the api, as well as those users being processed in order. If the internals of the api processed futures in parallel and didn't guarantee order, the result could be `List(enabledUser, dummy2User)` or `List(dummy2User, enabledUser)`.

It's working now and seems stable, just pointing this out as potential fragility down the line. In cases like this I tend to convert everything to a `Set` and use those.